segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -246,6 +249,80 @@ private static void validateContainerIds(Map<Integer, DebugStreamSegmentContaine
                 ex -> ex instanceof StreamSegmentNotExistsException, null);
     }
 
+
+    /**
+     * Copies the contents of container metadata segment and its attribute segment to new segments. The new names to the segments
+     * are passed as parameters.
+     * @param storage                    A {@link Storage} instance where segments are stored.
+     * @param containerId                A Container Id to get the name of the metadata segment.
+     * @param backUpMetadataSegmentName  A name to the back up metadata segment.
+     * @param backUpAttributeSegmentName A name to the back attribute segment.
+     * @param executorService            A thread pool for execution.
+     * @return                           CompletableFuture which when completed will have the segments' contents copied to another segments.
```

Comment:
"will indicate the operation has completed"

(the future itself won't have anything in it)

Reply:
Done.